noobaa_sa/bucket.py

```diff
@@ -87,22 +90,65 @@ def delete(self, bucket_name, config_root=None):
         if config_root is None:
             config_root = self.config_root
         log.info(f"Deleting {bucket_name} Bucket from NSFS")
-        cmd = f"sudo /usr/local/noobaa-core/bin/node {self.manage_nsfs} bucket delete --name {bucket_name} --config_root {config_root}"
+        cmd = f"{self.base_cmd} bucket delete --name {bucket_name} --config_root {config_root}"
         retcode, stdout, stderr = self.conn.exec_cmd(cmd)
         if retcode != 0:
             raise e.BucketDeletionFailed(f"Deleting bucket failed with error {stderr}")
-        
         log.info(stdout)
         log.info("Bucket deleted successfully")
 
 
-    def update(self, bucket_name):
+    def update(self, bucket_name, config_root=None, **kwargs):
         """
         Bucket update
 
         Args:
             bucket_name (str): Bucket name to be updated
+            config_root (str): Path to config root
+        """
```

Comment:
Please elaborate in the docstring on the update options and how to use them:
```suggestion
        """
        Bucket update

        Args:
            bucket_name (str): Bucket name to be updated
            config_root (str): Path to config root
            
        Supported update options via kwargs (based on https://url.corp.redhat.com/dd8609e):
            new_name (str): Update the bucket name
            email (str): Update the bucket owner email
            path (str): Update the bucket path
            bucket_policy (str): Update bucket policy with the valid JSON policy, or unset with ""
            fs_backend_type (str): Update filesystem type to "GPFS", CEPH_FS", "NFSv4" or or unset with ""

        Example usage:
            bucket_manager.update(bucket_name, new_name=new_bucket_name, path=new_bucket_path)
        """
```

Reply:
Thanks for the suggestion, Updated the code.